steps/firebase-app-distribution/0.1.0/step.yml

```diff
@@ -0,0 +1,108 @@
+support_url: https://github.com/guness/bitrise-step-firebase-app-distribution
+published_at: 2019-10-01T17:14:37.646878+03:00
+source:
+  git: https://github.com/guness/bitrise-step-firebase-app-distribution.git
+  commit: 0d95b3ad204416fad469d8afd8ebb1ba4848bc86
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- firebase_token: null
+  opts:
+    description: |
+      Firebase CI access token. You can create it by `firebase login:ci` on your local machine.
+
+      It is something in the form of `1/a1gqPSI3ryNcsDiZnCaHnY-hg9u9nJeosz3iNPe-S-RLdmqdypiJMOjoeOhOGsWK`
+    is_required: true
+    is_sensitive: true
+    title: Firebase Token
+- app_path: $BITRISE_IPA_PATH|$BITRISE_APK_PATH
+  opts:
+    description: |-
+      Path of IPA or APK file to deploy.
+
+      Default values:
+      iOS: `$BITRISE_IPA_PATH`
+      Android: `$BITRISE_APK_PATH`
+    is_required: true
+    title: App Path
+- app: null
+  opts:
+    description: |
+      You can find the App ID in the Firebase console, on the [General Settings page](https://console.firebase.google.com/project/_/settings/general/).
+
+      It is something in the form of:
+
+      `1:1234567890:android:0a1b2c3d4e5f67890`
+    is_required: true
+    summary: null
+    title: Firebase App ID
+- opts:
+    description: |
+      Release notes for this build.
+    is_required: false
+    title: Release Notes
+  release_notes: null
+- opts:
+    description: |
+      Plain text file containing release notes for this build.
+    is_required: false
+    title: Release Notes File
+  release_notes_file: null
+- opts:
+    description: |
+      The email addresses of the testers you want to invite.
+
+      You can specify the testers as a comma-separated list of email addresses:
+
+      `ali@example.com, bri@example.com, cal@example.com`
+    is_required: false
+    summary: null
+    title: Testers
+  testers: null
+- opts:
+    description: |
+      Plain text file containing a comma-separated list of email addresses.
+    is_required: false
+    summary: null
+    title: Testers File
+  testers_file: null
+- groups: null
+  opts:
+    description: |
+      The tester groups you want to invite (refer to [Manage testers](https://firebase.google.com/docs/app-distribution/manage-testers)).
+      Groups are specified using group aliases, which you can look up in the Firebase console.
+
+      You can specify the groups as a comma-separated list:
+
+      `qa-team, trusted-testers`
+    is_required: false
+    summary: null
+    title: Test Groups
+- groups_file: null
+  opts:
+    description: |
+      Plain text file containing a comma-separated list of group names.
+    is_required: false
+    summary: null
+    title: Test Groups File
```

Comment:
please add an "Additional command flags" input, so the users are able to specify any extra flags they need (also the `_file` suffixed ones)

Reply:
done